codalab/worker/worker_run_state.py

```diff
@@ -407,6 +409,17 @@ def check_and_report_finished(run_state):
             )
 
         def check_resource_utilization(run_state):
+
+            try:
+                cpu_usage, memory_usage = docker_utils.get_container_stats_on_mac(
```

Comment:
It's not clear to me what on mac means here. Does this command only work on a Mac?

Reply:
The reason I name it is the existing way that queries stats of docker containers does not work on Mac. Ref: https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/docker_utils.py#L228. I created this method so that I can develop on my machine.